docx/section.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, sectPr):
         self._sectPr = sectPr
 
     @property
+    def header(self):
+        return "I am not a header"
+
+    @property
```

Comment:
* This code is unsupported by a unit test, so doesn't belong here yet. Also, it fools the acceptance test to thinking you're further along than you are. So this bit is out for now